tools/commands/deploy.py

```diff
@@ -37,15 +35,17 @@ def __connect__(self):
         self.client.load_system_host_keys()
         self.client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
         logger.info("connect SSH {}@{}".format(self.proxy['user'], self.proxy['ip']))
-        self.client.connect(self.proxy['ip'], port=22, username=self.proxy['user'], password=self.proxy['pass'])
+        self.client.connect(self.proxy['ip'], port=22,
+                            username=self.proxy['user'], password=self.proxy['pass'])
 
     def run(self, commands=[]):
         ''' Run commands on target via proxy '''
         for c in commands:
             logger.debug("@{} --> {}".format(self.proxy['ip'], c))
             stdin, stdout, stderr = self.client.exec_command(c)
             if stdout.channel.recv_exit_status():
-                logger.error("Command failed:\n{}\nexit code - {}, stderr - {}".format(c, stdout.channel.recv_exit_status(), stderr.read()))
+                logger.error("Command failed:\n{}\nexit code - {}, stderr - {}".format(
```

Comment:
Maybe we should decide on a coding style for this kind of situation...

```
        "blahblahblah".format(
            x, y, z)
```

or

```
        "blahblahblah".\
            format(x, y, z)
```

Both are a bit ugly, but I have a slight preference for the second option.

Well, actually, I'd really prefer to use f-strings which would remove the need for ever using `"...".format()`, but apparently not everyone has python 3.6 :-(